services/ui-src/src/components/forms/SaveReturnButton.tsx

```diff
@@ -12,9 +13,18 @@ export const SaveReturnButton = ({
   onClick,
   submitting = false,
 }: Props) => {
+  const { userIsAdmin, userIsReadOnly } = useStore().user ?? {};
```

Comment:
suggestion: we could pull `userIsEndUser` and then check `!userIsEndUser`. if only because if we ever change or add user groups it might affect this logic